Documentation/Administration/BackendUsers/Administrator.rst

```diff
@@ -0,0 +1,146 @@
+and create a new administrator there.
+
+..  figure:: /Images/ManualScreenshots/BackendUser/CreateUser.png
+    :alt: Screenshot of the "Backend Users" module demonstrating the location of the "Create new backend user" button in the module header
+
+    Open the module :guilabel:`System > Backend Users` and click on "Create new backend user"
+
+When creating the user check the "Admin" option:
+
+..  figure:: /Images/ManualScreenshots/BackendUser/CreateAdmin.png
+    :alt: The Admin option is the first option in the tab "General" in the backend user edit form
+
+    When you check the Admin option the form needs to reload once. Answer the dialogue with "OK".
+
+You can :ref:`Grant System Maintainer rights <backend-users-system-maintainer>`
+in the Admin Tools later on.
+
+..  _backend-users-admin-install-tool:
+
+Using the Install Tool to create an administrator
+-------------------------------------------------
+
+Access the Install Tool at `https://my-site.ddev.site/typo3/install.php` using
+the Install Tool password defined during the installation process.
+
+..  figure:: /Images/ManualScreenshots/BackendUser/InstallTool.png
+    :alt: The Install Tool: Create Administrative User, Create new administrative users and grant them system maintainer privileges (optional).
+
+    In the module "Maintenance" module use the button "Create Administrator".
+
+If they should be a System Maintainer check the box.
+
+..  _backend-users-system-maintainer:
+
+Granting System Maintainer rights
+=================================
+
+..  note::
+    You must be logged in with a System Maintainer account or use the
+    Install Tool to grant System Maintainer rights. In
+    `Application context Development <https://docs.typo3.org/permalink/t3start:application-context>`_
+    all backend administrators can use the module.
+
+Using the module :guilabel:`Admin Tools > Settings` and card
+"Manage System Maintainers Access" you can manage which administrator accounts
+should be granted "System Maintainer" rights.
+
+..  figure:: /Images/ManualScreenshots/BackendUser/ManageSystemMaintainers.png
+    :alt: "Manage System Maintainers" window in the Admin Tools.
+
+    Disabled backend admins are marked with [DISABLED], you can choose them any way.
+
+On saving the changes are written to file :file:`config/system/settings.php`
+therefore this file needs to be writable.
+
+If your installation also has a file called :file:`config/system/additional.php`
+the settings can be overridden here. In this case changes you make in the
+Admin Tools do not take effect.
+
+A setting overriding the System Maintainer list could look like this:
+
+..  code-block:: php
+    :caption: config/system/additional.php
+
+    $GLOBALS['TYPO3_CONF_VARS']['SYS']['systemMaintainers'] => [
+        1,
+        3,
+        42
+    ];
+
+This setting is also documented in
+:ref:`systemMaintainers <t3coreapi:confval-globals-typo3-conf-vars-sys-systemmaintainers>`.
```

Comment:
Maybe we should a further section. If you want add administrators to be system maintainers it is NOT enough to just remove all user UIDs from array. As long as that value is an array ONE selected BE users are system maintainers. Either this value has to be removed completely and NULL from DefaultConfiguration will be used, or you have to set this value to NULL explicit.

Reply:
I would suggest to document this in TYPO3 explained this section is on how to make a system maintainer, not on details on how to avoid one. Would you like to make a PR there?